### ISSUE TITLE
Debouncing

### ISSUE THREAD

**Issue by dbrgn** [coredump-ch/ferraris-enumerator#3]
Can we have some debouncing implemented, meaning that the value is only sent if it didn't change for 10 seconds?

**Comment by dns2utf8** [comment]
No it responds instantly.

**Comment by dbrgn** [comment]
Exactly, that's the problem :) At least as long as we don't use grey coding.

**Comment by rnestler** [comment]
debouncing is always a good thing, regardless of coding, since every single switch change will bounce for several milliseconds. But a debouncing of ~5s would make sense.

**Comment by dbrgn** [comment]
Here's an example where a higher than intended value was submitted during toggling:

![img](https://tmp.dbrgn.ch/screenshots/20160928113756-7qhindih.png)

**Comment by fmorgner** [comment]
Valid points
